Evaluate-ModelarDB-Changes/main.py

```diff
@@ -261,18 +263,37 @@ def finish_output_file_and_kill_process(output_file):
         # Measure ingestion time in seconds.
         modelardbd = start_modelardbd(modelardb_folder, data_folder)
         ingestion_time = ingest_test_data(utilities_loader, test_data)
-        success = send_sigint_to_process(modelardbd)  # Ensure data is flushed.
-        if not ingestion_time or not success:
+        if not ingestion_time:
             print("ERROR: failed to ingest test data.")
             print_separator(current_change, last_change)
             continue
 
+        # Measure compaction time in seconds.
+        with tempfile.NamedTemporaryFile("w+") as small_select_query:
+            small_select_query.write(f"SELECT * FROM {TABLE_NAME} LIMIT 1")
+            compaction_time = execute_queries(small_select_query.name)
+            if not compaction_time:
+                print("ERROR: failed to compact files.")
+                print_separator(current_change, last_change)
+                continue
+
         # Measure query time in seconds.
-        modelardbd = start_modelardbd(modelardb_folder, data_folder)
-        query_time = execute_queries(queries)
-        success = send_sigint_to_process(modelardbd)  # Ensure process is gone.
-        if not query_time or not success:
-            print("ERROR: failed to execute queries.")
+        query_execution_times = {}
+        for query_set in query_sets:
+            query_time = execute_queries(query_set)
+            if not query_time:
+                print(f"error: failed to execute queries in {query_set}.")
```

Comment:
Very minor but in most places `ERROR:` is used instead. Maybe change this so it is consistent.

Reply:
Fixed.